src/App.js

```diff
@@ -8,8 +10,10 @@ const article = {
   date: 'August 30, 2019',
 };
 
-const App = () => (
-  <h1>React Article</h1>
+export const App = () => (
+  <Article
+    title={article.title}
+    text={article.text}
+    date={article.date}
+  />
```

Comment:
```suggestion
  <Article article={article} />
```
Можна передати цілий об'єкт, а не окремі властивості